### DISCUSSION TITLE
vite-config-alias-to-void-long-relative-path/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#71] [Announcements]
# vite-config-alias-to-void-long-relative-path/

利用 Vite alias 别名导入，可以在嵌套层次比较深的组件中，避免导入外层其他目录的组件，需要编写很长的相对路径的问题。

https://zxuqian.cn/vite-config-alias-to-void-long-relative-path/

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。
